Add citation and reference for DPT model (#59)
SforAiDl/vformer: docs/api/models/dense/dpt.rst

.. automodule:: vformer.models.dense.dpt
    :members:
    .. note::

>         Authors of this paper have done experiment with `vit_large` and vit_base` models with input image of size (384,384).

Reviewer: ```suggestion
        Authors of this paper experimented with `vit_large` and vit_base` models with input image of size (384,384).
```

>         Here we have extended a similar architecture for `vit_tiny` and flexible input size. Do consider this before using the model.

Reviewer: ```suggestion
        Here we have extended a similar architecture for `vit_tiny` which supports a variety of input sizes. 
```
